### PR TITLE
Add Documentation for Manual Install Process

### PR DESCRIPTION
**Pull request by Neumann-Nils**
### Description
This PR adds a README for a manual installation process (without Tiller), as demanded in #129.

### Checklist
Before submitting this PR, please make sure:
- [x] you have documented new or changed features

### CODE REVIEW

File: docs/manualInstallation.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Install karydia by hand
+It is possible to install karydia manually without the need of Helm and Tiller. This process is quite more complex and needs patience. If you just want to install karydia have a look at the [installation with Helm](../install/README.md).
```
Comment (ionysos) [inline docs/manualInstallation.md]: but Helm is still needed (locally) ... only the server part `Tiller` isn't needed anymore

File: install/README.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ karydia can be installed as a webhook into any Kubernetes cluster.
 
 The installation processes uses [Helm](https://github.com/helm/helm) to get karydia up and running and is split up into two parts.
 
+You can also do an [installation by hand](../docs/manualInstallation.md) without Helm and Tiller.
```
Comment (ionysos) [inline install/README.md]: Helm is still needed (locally) ...

File: docs/manualInstallation.md (hunk 1 above)
Comment (Neumann-Nils) [reply docs/manualInstallation.md]: Removed the "without helm" part.

File: install/README.md (hunk 2 above)
Comment (Neumann-Nils) [reply install/README.md]: Removed the "without helm" part.

File: docs/manualInstallation.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+```
+kubectl apply -f manifests/karydia/templates/post-install-webhook.yaml
+```
+
+## Make sure that karydia is running
+Take a quick look at the logs to see if karydia started:
+```
+kubectl logs -n kube-system $(kubectl get pods -n kube-system -l app=karydia -o jsonpath='{.items[0].metadata.name}') -f -c karydia
+
+time="2018-11-09T10:47:50Z" level=info msg="Listening on 0.0.0.0:33333"
```
Comment (ionysos) [inline docs/manualInstallation.md]: The new logger wrapper causes logs to appear in a different format which would result in:
```
{"component":"server","level":"info","msg":"[INFO] Listening on 0.0.0.0:33333","time":"2019-08-22T10:47:50Z"}
```

Comment (Neumann-Nils) [reply docs/manualInstallation.md]: Updated in both files.

**Pull request merged**